### ISSUE TITLE
Show dialog on destructive commands

### ISSUE THREAD

**Issue by dzikowski** [hyperledger-labs/fablo#324]
Fablo allows to call destructive commands on the network: `down`, `prune`, `recreate` etc. They should prompt if use is sure of doing it to avoid unwanted loss of the network state. It should be also allow to bypass the prompt with `--silent` flag, like `./fablo recreate --silent`

**Comment by Sanket-0510** [comment]
I think I understood the issue, I would like to give it a try!

**Comment by Sanket-0510** [comment]
Hi @dzikowski I am ready with my code. Is there any way through which I can build the fablo binary through source code and test the commands?

**Comment by dzikowski** [comment]
Yes! You can use `./fablo-build.sh` which builds local docker image with fablo distribution, and then use the Fablo script from source code `./fablo.sh init`, `./fablo.sh up` etc. 

Also, once you're done with code changes, you will also need to run `npm run test:e2e-update` to update our tests, and commit changes.
